website/pages/docs/mutations-and-input-types.mdx

```diff
@@ -156,7 +156,8 @@ const schema = new GraphQLSchema({
         },
         resolve: (_, { input }) => {
           // Create a random id for our "database".
-          const id = require('crypto').randomBytes(10).toString('hex');
+          import { randomBytes } from 'crypto';
```

Comment:
This isn’t valid here? Need to `await import()`

Reply:
Darn it :joy: